content/v0.75.2.md

```diff
@@ -0,0 +1,129 @@
+- [macOS](/downloads/macos/)
+
+## Table of Contents
+
+- [1. Summary](#1-summary)
+- [2. New Features in Detail](#2-new-features-in-detail)
+  - [2.1. Select Monochrome Palette Colour](#2-1-select-monochrome-palette-colour)
+  - [2.2. Raw Mouse Input](#2-2-raw-mouse-input)
+- [3. Bugfixes](#3-bugfixes)
+- [4. Notes for Packagers](#4-notes-for-packagers)
+- [5. Contributors](#4-contributors)
+- [6. Changelog](#5-changelog)
+
+## 1. Summary
+
+This is a maintenance release from 0.75.x stable branch - it includes a few
+changes and bugfixes backported from upcoming 0.76.x version.
+
+## 2. New Features in Detail
+
+### 2.1. Select Monochrome Palette Colour
+
+To be used with `machine = hercules` or `machine = cga_mono`. Users can
+now change the default monochrome palette colour (white, paperwhite, green,
+or amber) in a configuration file:
+
+```
+[render]
+monochrome_palette = amber
+```
+
+See: [#607](https://github.com/dosbox-staging/dosbox-staging/pull/607)
+
+
+### 2.2. Raw Mouse Input
+
+User can enable this setting to bypass operating system's mouse
+acceleration and sensitivity settings. This works in
+fullscreen or when the mouse is captured in window mode.
+
+Option can be changed in the sdl section of a configuration file:
+
+```
+[sdl]
+raw_mouse_input = true
+```
+
+This option is disabled by default, but some games (e.g. FPS titles,
+like DOOM or Dark Forces) will work better with raw input enabled
+(especially when used with modern, very sensitive gaming mouses).
+
+See: [#636](https://github.com/dosbox-staging/dosbox-staging/pull/636)
+
+
+## 3. Bugfixes
+
+- Bundle C++ Runtime libraries with Windows builds for users
+  on Windows 7, 8.x, and those who disabled Windows 10 update mechanism.
+  ([#675](https://github.com/dosbox-staging/dosbox-staging/pull/675))
+- Reimplement CLS command to prevent text mode reset.
+  ([#639](https://github.com/dosbox-staging/dosbox-staging/pull/639))
+- Fix whitespace handling in modem ATD command.
+  ([#652](https://github.com/dosbox-staging/dosbox-staging/pull/652))
+- Fix boot failure when using HDD image.
+  ([#522](https://github.com/dosbox-staging/dosbox-staging/pull/522))
+
+
+## 4. Notes for Packagers
+
+This release includes numerous fixes affecting our build system and other
+changes relevant for packaging (contributed by Gentoo packagers, and people
+preparing Debian, Fedora, and Flatpak packages) - prompt upgrade is recommended.
+
+If you're packaging DOSBox Staging for your OS, we have few tips:
+
+- We strongly suggest to keep our default binary name: `dosbox`.
+  DOSBox&nbsp;Staging is a drop-in replacement for older DOSBox versions - there
+  are programs depending on `dosbox` binary name (e.g. Wine or GUI frontends).
+  It's also the default name of our man page and appears in the documentation.
+- You don't need to generate your own desktop entry, appstream metadata
+  (metainfo.xml), nor package description - relevant files are in
+  `contrib/linux/` directory.
```

Comment:
(minor adjustment)
- You don't need to generate your own desktop entry, appstream metadata (metainfo.xml), **or** package description: please see the relevant files in the `contrib/linux/` directory.

Reply:
I thought "nor" is the preferred one in the "negative" context? 

I applied fix to the rest of the sentence.

Reply:
When you have a negative introduction, _"you don't need to ..."_ I think of that as a -1 multiplied by everything in the subsequent list, which is simply a, b, and c; or a, b, or c.

When you start your list (or sentence) with "neither" then "nor" carries the negative through to the end.  _Neither option A nor B were suitable._ 

The "neither" can be implied:  _I prefer not to compile with Clang, nor do I like to compile with VisualStudio._ If we rephrased it slightly then neither comes into play: _I prefer to compile with neither Clang nor Visual Studio._